packages/react-kit/src/components/test.page.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
```

Comment:
Since it is crucial part of ours applications I definitely want to have some unit tests for this HOC

Reply:
Yeah, this is the a final PR, just a POC to discuss API